### PR TITLE
Module 1: Project 3 - Contact Form

### PR DESCRIPTION
**Pull request by PraisesPJMT**
# Project 3: Contact Form

## Project Requirements
- Made sure that:
  - Name field accepts up to 30 chars.
  - Email field accepts only emails.
  - Message field accepts up to 500 chars.
  - All fields have placeholder text.
  - All fields have the correct type.
  - A user cannot submit the form if one or more of the fields is empty.
- Collect data using my account from the [Formspree service](https://formspree.io/html).
- In order to lay out the elements on the page,  I  used Flexbox.
- I stuck to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- I implemented all button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -148,19 +170,43 @@ <h2>About</h2>
...
+                <h2 class="contact-header">Get started</h2>
+                <p>
+                    I am interested in remote & freelance opportunities - especially ambitious or large projects. However, if you have other request or question, don't hesitate to use the form.
+                </p>
+                <form action="https://formspree.io/f/xlezygjj" method="POST">
+                    <input type="text" placeholder="Your name" maxlength="30" required="required" />
+                    <input type="email" placeholder="Your email" required="required" />
+                    <textarea id="message" cols="30" rows="10" maxlength="500" required="required" placeholder="Enter your message here..."></textarea>
+                    <button type="submit" class="button dark-button contact-button">Start collaboration</button>
+                </form>
```
Comment (HAMMAS-SALEEM) [inline index.html:186]: ### Your Form Response

formspree-error

- [ ]  Your form is not working. Please fix this by adding **name** attributes in your input fields. :+1:

File: index.html [hunk 2]
```diff
@@ -148,19 +170,43 @@ <h2>About</h2>
...
+            <a href="#" class="social-network-icon"><i class="fa-brands fa-twitter"></i
+        ></a>
+            <a href="#" class="social-network-icon"><i class="fa-brands fa-linkedin"></i
+        ></a>
+            <a href="#" class="social-network-icon"><i class="fa-brands fa-facebook-f"></i
+        ></a>
+            <a href="#" class="social-network-icon"><i class="fa-brands fa-instagram"></i
+        ></a>
         </div>
     </footer>
```
Comment (HAMMAS-SALEEM) [inline index.html:209]: ### You Footer
footer (copy)
### Figma Footer
Figma

- [ ] You can see that in your design, you have margins on left and right sides. Please remove these margins :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
## Required Changes :recycle:

### Hey Team (@PraisesPJMT, @mj-e-boc, @kessio) :wave:

### Highlights :1st_place_medal:

Linters are green :heavy_check_mark: 
Good implementation of validations :heavy_check_mark:
Most Design is very good :heavy_check_mark:


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @HAMMAS-SALEEM in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -148,19 +170,43 @@ <h2>About</h2>
                 </div>
             </div>
         </section>
+        <!-- CONTACT FORM -->
+        <section class="contact-form">
+            <div class="contact-form-background"></div>
+            <div class="contact-form-container">
```
[the remark this answers is not in the document]
Comment (PraisesPJMT) [reply index.html:176]: Oh Yes! I set a fixed width for the whole "body" that's why margins get to appear at the edges.
I just removed that.

I also did not notice the background colour for the "Contact Section".
I just made the necessary corrections.

Thanks so much.

File: index.html (hunk 1 above)
Comment (PraisesPJMT) [reply index.html:186]: Oh yes! We discussed this during the paired programming session. We totally did not include it. 
We just made the additions.
Thanks.

File: index.html (hunk 2 above)
Comment (PraisesPJMT) [reply index.html:209]: The footer margin was also affected by the fixed-width attribute of the "body".
We just made the necessary corrections.

Thanks:

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**